src/js/pages/jobs/JobDetailPage.js

```diff
@@ -10,6 +10,8 @@ import mixin from "reactjs-mixin";
 import JobFormModalContainer from "#PLUGINS/jobs/src/js/JobFormModalContainer";
 import TaskStates from "#PLUGINS/services/src/js/constants/TaskStates";
 import JobsBreadcrumbs from "#PLUGINS/jobs/src/js/components/JobsBreadcrumbs";
+import menuGetActions from "#PLUGINS/jobs/src/js/JobsMenu";
```

Comment:
if using default export, please dont rename it from Filename :(

Reply:
Do you know why it is a good practice not to rename it?

Reply:
It is what we do in the codebase, using default and calling it the same as the file 🤷‍♂️  Also there is the Rule that one should only declare one component per file.

Reply:
I am just curious about the why. I will rename anyway.

Reply:
OCD 😅